ast/misc.go

```diff
@@ -109,7 +109,6 @@ type DeallocateStmt struct {
 	stmtNode
 
 	Name string
-	ID   uint32 // For binary protocol, there is no Name but only ID.
```

Comment:
`Session.DropPreparedStmt(stmtID uint32)` will be called, and we don't need to create a `ast.DeallocStmt` to do the work, just delete the statement in session variable.